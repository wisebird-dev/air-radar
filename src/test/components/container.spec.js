import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Container from '../../components/container'

test('Container renders correctly', async () => {
  const tree = await renderer.create(<Container />).toJSON()

  expect(tree).toMatchSnapshot()
})