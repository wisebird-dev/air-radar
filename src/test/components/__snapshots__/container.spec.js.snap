// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  margin: 0 auto 30px;
  max-width: 50em;
  padding-left: 15px;
  padding-right: 15px;
}

<div
  className="c0"
/>
`;
